src/ai/embeddings.rs

```diff
@@ -108,44 +117,60 @@ pub async fn get_related_full_docs(
     client: &Client<OpenAIConfig>,
```

Comment:
Not related to this specific story, but is `get_related_full_docs` even used anywhere? I could not find any reference to it.

Reply:
Yes, indeed it's currently no longer used. I'll open a separate PR to remove it.